cs-property-service/src/main/java/org/egov/cpt/models/CorrespondenceAddress.java

```diff
@@ -0,0 +1,65 @@
+package org.egov.cpt.models;
```

Comment:
New model is not required. These fields are nothing but the fields from Address.